components/create-collective/sections/ConnectGithub.js

```diff
@@ -99,7 +99,10 @@ class ConnectGithub extends React.Component {
   };
 
   getGithubConnectUrl() {
-    const urlParams = new URLSearchParams({ redirect: `${getWebsiteUrl()}/create/v2/opensource` });
+    const urlParams = new URLSearchParams({
```

Comment:
Interesting - what is context in this case?

Reply:
It's to help the API knows what we're trying to do with GitHub, then tweak the permissions based on that: https://github.com/opencollective/opencollective-api/pull/3440